src/components/data-processing/DataIntegration/CalculationConfig.jsx

```diff
@@ -186,11 +186,10 @@ const CalculationConfig = (props) => {
                   { dimensionalityReduction: { excludeGeneCategories: val } },
                 )}
                 value={dimensionalityReduction.excludeGeneCategories}
-                disabled
               >
                 <Space direction='vertical'>
-                  <Checkbox value='ribosomal'>ribosomal</Checkbox>
-                  <Checkbox value='mitochondrial'>mitochondrial</Checkbox>
+                  <Checkbox disabled value='ribosomal'>ribosomal</Checkbox>
+                  <Checkbox disabled value='mitochondrial'>mitochondrial</Checkbox>
                   <Checkbox value='cellCycle'>cell cycle</Checkbox>
```

Comment:
doesn't adding disabled actually disable them?

Reply:
They were disabled as a block (check the original file, I removed the disabled there). For the moment we only want/need to enable the cell cycle one.